cmd/core/pfcp_handlers.go

```diff
@@ -86,14 +98,21 @@ func HandlePfcpAssociationSetupRequest(conn *PfcpConnection, msg message.Message
 	conn.NodeAssociations[addr] = remoteNode
 	log.Info().Msgf("Saving new association: %+v", remoteNode)
 
+	featuresOctets := [41]uint8{}
+	if config.Conf.FeatureFTUP {
+		setBitUP(featuresOctets[:], 4) // FTUP
+	}
+	if config.Conf.FeatureUEIP {
+		setBitUP(featuresOctets[:], 18) // UEIP
```

Comment:
Why do you thing that this method is simpler? According to 3GPP specification, we have octets and feature bits in specific octet, but not an end-to-end bit numbering